test/test_helper.rb

```diff
@@ -12,6 +12,8 @@
 require 'action_mailer'
 
 ExceptionNotifier.testing_mode!
+require 'support/exception_notifier_helper'
```

Comment:
The helper method, `ExceptionNotifier#.reset_notifiers!`, is available in any test cases, but currently used in only two test files that are relevant - `test/exception_notifier_test.rb` and `test/exception_notification/rack_test.rb`.